Windows/GPU/D3D11Context.cpp

```diff
@@ -98,6 +111,19 @@ void D3D11Context::Resize() {
 }
 
 void D3D11Context::Shutdown() {
+#ifdef _DEBUG
+	d3dInfoQueue_->SetBreakOnSeverity(D3D11_MESSAGE_SEVERITY_CORRUPTION, false);
+	d3dInfoQueue_->SetBreakOnSeverity(D3D11_MESSAGE_SEVERITY_ERROR, false);
+	d3dInfoQueue_->SetBreakOnSeverity(D3D11_MESSAGE_SEVERITY_WARNING, false);
+#endif
+#ifdef _DEBUG
+	d3dDebug_->ReportLiveDeviceObjects(D3D11_RLDO_SUMMARY | D3D11_RLDO_DETAIL);
+#endif
+	d3dDebug_->Release();
+	d3dInfoQueue_->Release();
```

Comment:
Guessing those two should be included under _DEBUG above. It also seems to be repeated without reason, I mean the ifdef.